src/suricata-common.h

```diff
@@ -459,6 +459,7 @@ typedef enum {
     LOGGER_JSON_TEMPLATE_RUST,
```

Comment:
`src/tests/fuzz/confyaml.c` should also be modified

Reply:
Should something about this also be added to the setup-app-layer.py script? What do you think, @jasonish ?

Reply:
I wasn't aware of this file.  @catenacyber What conditions would require this file to be updated? The additional/remove of any parser?

Reply:
> I wasn't aware of this file. @catenacyber What conditions would require this file to be updated? The additional/remove of any parser?

Indeed.
I guess we could replace this part of the config by a util function like EnableAllProtocols...

Reply:
PS : I was not aware of setup-app-layer.py ;-p

Reply:
Hey @catenacyber, is adding the lines
```c
postgresql:
    enabled: yes
```
enough, or should I edit something else as well?